### ISSUE TITLE
no-inline-styles: Enable `allowDynamicStyles` option by default

### ISSUE THREAD

**Issue by Turbo87** [ember-template-lint/ember-template-lint#1044]
https://github.com/ember-template-lint/ember-template-lint/pull/672 introduced a new `allowDynamicStyles` option for the `no-inline-styles` rule. I would like to propose to make that the default for the v2.x release.

https://getbootstrap.com/docs/4.0/components/progress/ for example is using inline styles to set the width of a progressbar, I've seen other code use dynamic background colors for user avatars, etc.

tl;dr there are quite a few common valid use-cases for dynamic inline styles and we shouldn't be warning about **all** of them

**Comment by rwjblue** [comment]
@Turbo87 - It is not clear to me what you mean here, do you mean defaulting to `true` or `false`?

**Comment by Turbo87** [comment]
@rwjblue sorry, should have been clearer. I would want the default to change from `false` to `true`. in other words: it should **not** warn about dynamic styles by default.

**Comment by bmish** [comment]
I'll open a PR for this shortly.

**Comment by rwjblue** [comment]
Closed by #1052
